### PR TITLE
Transform default CommonJS imports

### PR DESCRIPTION
**Pull request by Mrtenz**
This adds a new helper function for default CommonJS imports. The helper function `importDefault` is based on `tsc`s helper function, and solves default imports from ESM of a CJS module which specifies `__esModule`. For example, the output of `tsc` for a CommonJS module with a default export may be something like this (reformatted):

```ts
const __importDefault =
  (this && this.__importDefault) ||
  function (mod) {
    return mod && mod.__esModule ? mod : { default: mod };
  };

Object.defineProperty(exports, '__esModule', { value: true });
exports.default = void 0;

const foo_1 = require('./foo');

Object.defineProperty(exports, 'default', {
  enumerable: true,
  get() {
    return __importDefault(foo_1).default;
  },
});
```

TypeScript sets `__esModule` to `true` and then exports the default value as `exports.default` rather than just `exports`. When importing this in ESM normally, this results in the following:

```ts
import foo from 'cjs-module';

console.log(foo); // { default: 'value' }
```

After this change, each default import from a CJS module will be changed to use the `importDefault` helper, so the resulting code looks something like this instead:

```ts
function $importDefault(module) {
  // ...;
}

import $foo from 'cjs-module';
const foo = $importDefault($foo);

console.log(foo); // 'value'
```

### CODE REVIEW

**Comment by rekmarks** [comment]
What was the previous behavior when importing ESM modules with default exports from CJS?

**Comment by Mrtenz** [comment]
> What was the previous behavior when importing ESM modules with default exports from CJS?

This only changes importing CJS modules from ESM. Currently this would happen (which is also in the PR description):

```
import foo from 'cjs-module';

console.log(foo); // { default: 'value' }
```

**Comment by rekmarks** [comment]
Ah, I see. Reading comprehension bad.

**Review by rekmarks — APPROVED** [review]
LGTM, with the caveat that I don't understand how `getDefaultImportTransformer()` works.

File: packages/cli/src/transformers.ts [hunk 1]
```diff
@@ -484,6 +486,115 @@ export function getImportMetaTransformer({ typeChecker }: TransformerOptions) {
...
+ * ```ts
+ * import foo from 'module';
+ * ```
+ *
+ * will be transformed to:
+ *
+ * ```ts
+ * import * as $helpers from '@ts-bridge/helpers';
+ * import $foo from 'module';
+ * const foo = $helpers.importDefault($foo);
```
Comment (FrederikBolding) [inline packages/cli/src/transformers.ts:504]: This should be updated

**Review by FrederikBolding — APPROVED** [review]

**Pull request merged**